dist_tables/append_distribution.rst

```diff
@@ -122,15 +122,13 @@ $$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
 
 The methods described above enable you to achieve high bulk load rates which are sufficient for most use cases. If you require even higher data load rates, you can use the functions described above in several ways and write scripts to better control sharding and data loading. For more information, you can consult the :ref:`scaling_data_ingestion` section of our documentation.
 
-Dropping Shards
+Expiring Data
 ---------------
 
 In append distribution, users typically want to track data only for the last few months / years. In such cases, the shards that are no longer needed still occupy disk space. To address this, Citus provides a user defined function master_apply_delete_command() to delete old shards. The function takes a `DELETE <http://www.postgresql.org/docs/9.5/static/sql-delete.html>`_ command as input and deletes all the shards that match the delete criteria with their metadata.
 
 The function uses shard metadata to decide whether or not a shard needs to be deleted, so it requires the WHERE clause in the DELETE statement to be on the distribution column. If no condition is specified, then all shards are selected for deletion. The UDF then connects to the worker nodes and issues DROP commands for all the shards which need to be deleted. If a drop query for a particular shard replica fails, then that replica is marked as TO DELETE. The shard replicas which are marked as TO DELETE are not considered for future queries and can be cleaned up later.
 
-Please note that this function only deletes complete shards and not individual rows from shards. If your use case requires deletion of individual rows in real-time, please consider using the hash distribution method.
```

Comment:
Let's not remove the entire paragraph as the first note is still valid. We could say something like:
"Please note that this function only deletes complete shards and not individual rows from shards. If your use case requires deletion of individual rows, please see the master_modify_multiple_shards UDF" (or something like that).